code/game/machinery/computer/communications.dm

```diff
@@ -1,6 +1,8 @@
 //This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
 
 var/const/CALL_SHUTTLE_REASON_LENGTH = 12
+var/const/COMMUNICATION_COOLDOWN = 600
+var/const/COMMUNICATION_COOLDOWN_AI = 600
 

```

Comment:
I can only do so with the COOLDOWN ones, because the shuttle call reason length is used in other places, namely:
https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/shuttles.dm#L192
https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/silicon/ai/ai.dm#L313